### PR TITLE
Laboratory work #8, Darya Krinochkina - 20FPL1

### PR DESCRIPTION
**Pull request by DashaKrinochkina**

### CODE REVIEW

File: lab_7_llm/main.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 from pathlib import Path
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: Checkout the files from 7 lab to the master

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+datasets==2.16.1
```
Comment (SalnikovIgor) [inline requirements.txt:1]: Add line `--extra-index-url https://download.pytorch.org/whl/cpu`

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_8_llm/main.py [hunk 3]
```diff
@@ -34,6 +29,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Delete. And add a check: `_raw_data` containt `None`? If yes, then rise error

File: lab_8_llm/main.py [hunk 4]
```diff
@@ -43,6 +39,15 @@ def obtain(self) -> None:
...
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        dataset = load_dataset(self._hf_name, split="test")
+        self._raw_data = dataset.to_pandas()
+
+        if not isinstance(self._raw_data, DataFrame):
+            raise TypeError
+
+    @property
+    def raw_data(self) -> DataFrame:
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Also delete

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_8_llm/main.py (hunk 3 above)
Comment (DashaKrinochkina) [reply lab_8_llm/main.py]: done

File: lab_8_llm/main.py (hunk 4 above)
Comment (DashaKrinochkina) [reply lab_8_llm/main.py]: done

**Pull request closed without merging**